src/main/java/pl/rejmanbeata/bakery/service/EmployeeService.java

```diff
@@ -4,29 +4,49 @@
 import org.springframework.stereotype.Service;
 import pl.rejmanbeata.bakery.database.EmployeeEntity;
 import pl.rejmanbeata.bakery.jpa_repository.EmployeeRepository;
+import pl.rejmanbeata.bakery.mapper.EmployeeMapper;
+import pl.rejmanbeata.bakery.model.employee.Employee;
 
 import java.util.List;
+import java.util.Optional;
 
 @Service
 @AllArgsConstructor
 public class EmployeeService {
-
     private final EmployeeRepository employeeRepository;
+    private final EmployeeMapper employeeMapper;
 
-    public EmployeeEntity save(EmployeeEntity address) {
-        return employeeRepository.save(address);
+    public Employee save(Employee employee) {
+        EmployeeEntity employeeEntity = employeeMapper.employeeToEmployeeEntity(employee);
+        EmployeeEntity savedEmployee = employeeRepository.save(employeeEntity);
+        return employeeMapper.employeeEntityToEmployee(savedEmployee);
     }
 
-    public List<EmployeeEntity> getAllEmployees() {
-        return employeeRepository.findAll();
+    public List<Employee> getAllEmployees() {
+        return employeeRepository.findAll()
+                .stream()
+                .map(employeeMapper::employeeEntityToEmployee)
+                .toList();
     }
 
-    public EmployeeEntity getEmployeeById(Long id) {
-        return employeeRepository.findById(id).orElse(null);
+    public Employee getEmployeeById(Long id) {
+        return employeeMapper.employeeEntityToEmployee(employeeRepository.findById(id).orElse(null));
```

Comment:
I think we can just call `findById(id).get()` (instead of `orElse(null)` since it will have the same result. Or?